contrib/kube-prometheus/docs/developing-alerts-and-dashboards.md

```diff
@@ -2,40 +2,222 @@
 
```

Comment:
How about renaming this file to `developing-prometheus-rules-and-grafana-dashboards.md` as it includes the recoding rules docs as well?